test/scripts/downloadChainSpec.ts

```diff
@@ -49,7 +49,7 @@ yargs(hideBin(process.argv))
                 }
                 process.stdout.write(`Done ✅\n`);
                 const onChainGenesisData = await api.createType(
-                    "TpContainerChainGenesisDataContainerChainGenesisData",
+                    "DpContainerChainGenesisDataContainerChainGenesisData",
```

Comment:
Remember to mention this in breaking changes, this will probably break all the registration scripts we have

Reply:
why so?

Reply:
yes but I dont think we use it

Reply:
maybe the dapp does